applications/pelion_client/README.rst

```diff
@@ -0,0 +1,560 @@
+   The Device Management Update link points to the documentation of the version compatible with the |NCS|.
+   For details, refer to the `Pelion Device Management documentation`_ and `Pelion Device Management release notes`_.
+
+User interface
+**************
+
+The application uses buttons for updating the OMA object resource counters in Pelion Device Management Portal.
+It also uses LEDs to reflect the application tasks.
+
+Buttons
+=======
+
+The application uses the following buttons on the supported development kits:
+
+* **Button 1** -- Connected with the OMA Digital Input object instance 0.
+* **Button 2** -- Connected with the OMA Digital Input object instance 1.
+* **Button 3** -- Connected with the OMA Digital Input object instance 2.
+* **Button 4** -- Connected with the OMA Digital Input object instance 3.
+
+Pressing each of these buttons updates the respective object's state resource.
+Each button press is counted and the count value is reflected onto the respective object's counter resource.
+
+You can reset Digital Input objects' counters by putting zero into their respective values in the Pelion Device Management Portal.
+
+.. note::
+   |factory_reset_note|
+
+LEDs
+====
+
+The application displays LED behavior that corresponds to the task performed by the application.
+The following table shows the LED behavior demonstrated by the application:
+
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Status                        | LED behavior              | Related application state                                                      |
++===============================+===========================+================================================================================+
+| Network back end disconnected | **LED1** blinking         | NET_STATE_DISABLED or NET_STATE_DISCONNECTED                                   |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Pelion connection search      | **LED2** blinking         | PELION_STATE_DISABLED or PELION_STATE_INITIALIZED or PELION_STATE_UNREGISTERED |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Network back end connected    | **LED1** solid on         | NET_STATE_CONNECTED                                                            |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Pelion connection established | **LED2** solid on         | PELION_STATE_REGISTERED                                                        |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
```

Comment:
We need to update this table, as we introduced new state: `PELION_STATE_SUSPENDED` where the LED2 is breathing with 200ms period.

```
+-------------------------------+---------------------------------+--------------------------------------------------------------------------------+
| Status                        | LED behavior                    | Related application state                                                      |
+===============================+=================================+================================================================================+
| Network back end disconnected | **LED1** breathing 500ms period | NET_STATE_DISABLED or NET_STATE_DISCONNECTED                                   |
+-------------------------------+---------------------------------+--------------------------------------------------------------------------------+
| Network back end connected    | **LED1** solid on               | NET_STATE_CONNECTED                                                            |
+-------------------------------+---------------------------------+--------------------------------------------------------------------------------+
| Pelion connection search      | **LED2** breathing 500ms period | PELION_STATE_DISABLED or PELION_STATE_INITIALIZED or PELION_STATE_UNREGISTERED |
+-------------------------------+---------------------------------+--------------------------------------------------------------------------------+
| Pelion device suspended       | **LED2** breathing 200ms period | PELION_STATE_SUSPENDED                                                         |
+-------------------------------+---------------------------------+--------------------------------------------------------------------------------+
| Pelion connection established | **LED2** solid on               | PELION_STATE_REGISTERED                                                        |
+-------------------------------+---------------------------------+--------------------------------------------------------------------------------+
```